chefignore

```diff
@@ -104,3 +104,11 @@ Strainerfile
 ###########
 .vagrant
 Vagrantfile
+
+# GitHub #
+##########
+.delivery/project.toml
+.github/ISSUE_TEMPLATE.md
```

Comment:
You probably want to just use the directory here since at some point we'll add more stuff

Reply:
I didnt come up with this..